weave-js/src/components/PagePanelComponents/PublishModal.tsx

```diff
@@ -47,6 +48,29 @@ const iconStyle = {
   marginRight: 8,
 };
 
+const useLastVisitedPath = () => {
+  const [lastVisitedPath] = useLocalStorage<string>('lastVisited', '');
+  const [lastVisitedEntity, lastVisitedProject] = useMemo(() => {
+    let e = null;
```

Comment:
I haven't tested it, but I think this whole thing could be something simpler like:
`const [prefix, browse, scope, entityName, projectName] = lastVisitedPath.split('/')`

Reply:
nice! changed in next commit.